README.md

```diff
@@ -80,6 +80,7 @@ Add `Pomelo.EntityFrameworkCore.MySql` to the services configuration in your the
 using System;
 using Microsoft.EntityFrameworkCore;
 using Pomelo.EntityFrameworkCore.MySql.Infrastructure;
+using Pomelo.EntityFrameworkCore.MySql.Storage;
```

Comment:
Thinking about it, it is probably cleaner to just use another overload (which we did previously) in the sample code:

```c#
services.AddDbContextPool<YourDbContext>(options => options
    // replace with your connection string
    .UseMySql("Server=localhost;Database=ef;User=root;Password=1234;",
              mySqlOptions => mySqlOptions
                  // replace with your Server Version and Type
                  .ServerVersion(new Version(8, 0, 20), ServerType.MySql)
    )
);
```

We should/will consolidate some of the namespaces for .NET 5 and this way, people will not get compile time errors for this one.

Reply:
Whatever works. I just didn't want another developer to spend 15 minutes trying to figure out why the code in the example (being so simple) didn't compile.

Reply:
If you change it, I merge it.

Reply:
changed